cmd/kubeadm/app/cmd/alpha/certs_test.go

```diff
@@ -294,28 +294,30 @@ func TestRenewUsingCSR(t *testing.T) {
 	}
 }
 
-// A minimal kubeadm config with just enough values to avoid triggering
-// auto-detection of config values at runtime.
-const kubeadmConfig = `
-apiVersion: kubeadm.k8s.io/v1beta2
-kind: InitConfiguration
-localAPIEndpoint:
-  advertiseAddress: 192.0.2.1
-nodeRegistration:
-  criSocket: /path/to/dockershim.sock
----
-apiVersion: kubeadm.k8s.io/v1beta2
-kind: ClusterConfiguration
-certificatesDir: /custom/config/certificates-dir
-kubernetesVersion: v1.19.0
```

Comment:
e.g. from my previous comment, the PR adds this constant in one commit and then removes it in another.